amy/fiscal/tests/test_membership.py

```diff
@@ -865,9 +866,41 @@ def test_membership_extended(self):
             response, reverse("membership_details", args=[membership.pk])
         )
         membership.refresh_from_db()
-        self.assertEqual(membership.extended, 30)
+        self.assertEqual(membership.extensions, [30])
```

Comment:
Could you add a test where the membership has been extended more than once?

Reply:
Good idea. I'll add it.